### ISSUE TITLE
Help tooltip/popup after Autocomplete skips first line/title of documentation

### ISSUE THREAD

**Issue by ahcyip** [rstudio/rstudio#7839]
### System details
$mode
[1] "desktop"

$version
[1] ‘1.3.1093’

### Steps to reproduce the problem

RStudio's autocomplete and tooltip displaying some of the help file skips the title / first line of the documentation and instead starts displaying the text under "Usage" (about function arguments) and then skips to the content under "Description". Often, there is critical information about the function in the title, especially if the function name is somewhat ambiguous / not immediately intuitive. e.g. fct_reorder, fct_relevel, str_sub, ...

See the source pane for the tooltip (the Help tab is there just for reference of what I would want to see in the tooltip)
https://user-images.githubusercontent.com/5713411/89116051-86fd0f80-d45d-11ea-82e0-2b825a986e8d.png

### Expected
The tooltip would ideally include the title/first line to say 
"fct_reorder(.f, .x, .fun = median, ..., .desc = FALSE)" (the "usage" line)
"Reorder factor levels by sorting along another variable", (the title line)
before showing the next few lines "fct_reorder() is useful for 1d displays where ..." (then, the description contents)

Basically, not all function names are intuitive enough to skip the first line / title of the function documentation. (as discussed here https://github.com/tidyverse/forcats/issues/269)

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmcphers** [comment]
> $version
> [1] ‘1.2.5033’

Have you installed the latest version and confirmed that the issue still persists? It's 1.3.1093:

https://rstudio.com/products/rstudio/download/

**Comment by ahcyip** [comment]
Sorry, I have now updated to the actual latest. Issue persists. 

Summary: First line (title?) of documentation/help file is missing from the tooltip that pops up after autocomplete shows up (see screenshot). Not sure if this was intended, but in some cases such as fct_reorder, critical information is missing from the tooltip, making it not-very-helpful.

$mode
[1] "desktop"

$version
[1] ‘1.3.1093’

$release_name
[1] "Apricot Nasturtium"

**Comment by kevinushey** [comment]
I see the same behavior even in RStudio 1.0:

<img width="650" alt="Screen Shot 2020-10-06 at 10 28 41 PM" src="https://user-images.githubusercontent.com/1976582/95290949-53ef4780-0823-11eb-8986-2d9dee9f0f5b.png">

and the tooltip matches what you would see in the Help documentation:

<img width="1437" alt="Screen Shot 2020-10-06 at 10 28 00 PM" src="https://user-images.githubusercontent.com/1976582/95290957-5b165580-0823-11eb-9d8d-64b97a46f19e.png">

Can you clarify what you were expecting to see?

**Comment by ahcyip** [comment]
Looks like I imagined it being in an older RStudio version.
I was hoping to see the tooltip include the title "Reorder factor levels by sorting along another variable" since it's the most informative part of the documentation. 

Or is this a case of a Description section that needs editing (so that the "title" information is included)? If so, I've documented that issue there: https://github.com/tidyverse/forcats/issues/269

**Comment by ahcyip** [comment]
Here's a simpler example: 
If you start typing str_pad, you see that the help popup says:

str_pad(string, width, side = c("left", "right", "both"), pad = " ")
Vectorised over string, width and pad.

It seems ideally, you would want to see:
str_pad(string, width, side = c("left", "right", "both"), pad = " ")
Pad a string.
Vectorised over string, width and pad.

**Comment by hadley** [comment]
I think this would be a very useful usability improvement.

**Comment by ahcyip** [comment]
Sorry, I accidentally closed this and have reopened it.

But here's another example of how it's not great right now: 
I type str_su and the autocomplete pops up, but it's missing critical information that's in the title of the help documentation, which isn't being shown in the autocomplete pop-up, and instead, provides the body description, which may not make sense without the critical title information.

![image](https://user-images.githubusercontent.com/5713411/121441189-8e772a00-c946-11eb-85b2-fe69bdb371a6.png)

**Comment by ahcyip** [comment]
Checking in with another motivating example:
![image](https://user-images.githubusercontent.com/5713411/173995974-2a63f996-193e-44e8-b557-2eb4e4e6190a.png)

**Comment by ahcyip** [comment]
thanks, looks great!!
